examples/expressions/main.rs

```diff
@@ -0,0 +1,84 @@
+mod metadata;
+mod note;
+
+use gtk::gio;
+use gtk::glib;
+use gtk::prelude::*;
+
+use self::metadata::Metadata;
+use self::note::Note;
+
+fn main() {
+    let application = gtk::Application::new(
+        Some("com.github.gtk-rs.examples.expressions"),
+        Default::default(),
+    );
+
+    application.connect_activate(build_ui);
+    application.run();
+}
+
+fn build_ui(app: &gtk::Application) {
+    let window = gtk::ApplicationWindow::new(app);
+    window.set_title(Some("Expressions"));
+    window.set_default_size(300, 500);
+
+    let factory = gtk::SignalListItemFactory::new();
+    factory.connect_setup(|_, list_item| {
+        let hbox = gtk::Box::new(gtk::Orientation::Horizontal, 24);
+
+        let title_label = gtk::Label::new(None);
+        hbox.append(&title_label);
+
+        let last_modified_label = gtk::Label::new(None);
+        hbox.append(&last_modified_label);
+
+        // Instead of binding properties and manually unbinding them, you can create expressions.
+        // The value can be obtained even if it is several steps away.
+        let metadata_expression = list_item
+            .property_expression("item")
+            .chain_property(Note::static_type(), "metadata");
+
+        metadata_expression
+            .chain_property(Metadata::static_type(), "title")
+            .chain_closure(|args| {
+                let title: String = args[1].get().unwrap();
+                format!("Title: {}", title)
+            })
+            .bind(&title_label, "label", None::<&gtk::Widget>);
+
+        metadata_expression
+            .chain_property(Metadata::static_type(), "last-modified")
+            .chain_closure(|args| {
+                let last_modified: glib::DateTime = args[1].get().unwrap();
+                format!("Last Modified: {}", last_modified.format_iso8601().unwrap())
+            })
+            .bind(&last_modified_label, "label", None::<&gtk::Widget>);
```

Comment:
Same here
```suggestion
            .bind(&last_modified_label, "label", gtk::NONE_WIDGET);
```

Reply:
Why, they're equivalent

Reply:
Sure, but why have them at all if we don't even use them in our examples.